dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestControllerIT.java

```diff
@@ -425,5 +514,127 @@ private int getNumberOfPdfPages(byte[] content) throws IOException {
         }
     }
 
+    @Test
+    public void getBitstreamFormat() throws Exception {
+
+        getClient().perform(get("/api/core/bitstreams/" + bitstream.getID() + "/format"))
+                .andExpect(status().isUnauthorized());
 
+        getClient(getAuthToken(eperson.getEmail(), password))
+                .perform(get("/api/core/bitstreams/" + bitstream.getID() + "/format"))
+                .andExpect(status().isForbidden());
+
+        context.turnOffAuthorisationSystem();
+
+        resourcePolicy = resourcePolicyService.create(context);
+        resourcePolicy.setEPerson(eperson);
+        resourcePolicy.setAction(READ);
+        resourcePolicy.setdSpaceObject(bitstream);
+
+        context.commit();
+
+        getClient(getAuthToken(eperson.getEmail(), password))
+                .perform(get("/api/core/bitstreams/" + bitstream.getID() + "/format"))
+                .andExpect(status().isOk());
+
+        getClient(getAuthToken(admin.getEmail(), password))
+                .perform(get("/api/core/bitstreams/" + bitstream.getID() + "/format"))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$", matchBitstreamFormat(
+                        supportedFormat.getID(),
+                        supportedFormat.getMIMEType(),
+                        supportedFormat.getDescription(),
+                        supportedFormat.getShortDescription(),
+                        "SUPPORTED"
+                )));
+    }
+
+    @Test
+    public void updateBitstreamFormat() throws Exception {
+
+        getClient().perform(
+                put("/api/core/bitstreams/" + bitstream.getID() + "/format")
+                        .contentType(parseMediaType(TEXT_URI_LIST_VALUE))
+                        .content(
+                                "https://localhost:8080/spring-rest/api/core/bitstreamformat/" + knownFormat.getID()
+                        )
+        ).andExpect(status().isUnauthorized());
+
+        getClient(getAuthToken(eperson.getEmail(), password)).perform(
+                put("/api/core/bitstreams/" + bitstream.getID() + "/format")
+                        .contentType(parseMediaType(TEXT_URI_LIST_VALUE))
+                        .content(
+                                "https://localhost:8080/spring-rest/api/core/bitstreamformat/" + knownFormat.getID()
+                        )
+        ).andExpect(status().isForbidden());
+
+        context.turnOffAuthorisationSystem();
+
+        resourcePolicy = resourcePolicyService.create(context);
+        resourcePolicy.setEPerson(eperson);
+        resourcePolicy.setAction(WRITE);
+        resourcePolicy.setdSpaceObject(bitstream);
+
+        bitstream = context.reloadEntity(bitstream);
+        bitstreamService.addMetadata(context, bitstream, "dc", "format", null, null, "test format");
```

Comment:
Again, use a Builder here, rather than doing a `reloadEntity` and changing values.  Each test should be standalone, and if you change dc.format here, it will change it for other test methods (in this same class) too.